PolyEngine/Standalone/Src/Main.cpp

```diff
@@ -167,6 +196,32 @@ int main(int argc, char* args[])
 
 		// Engine loop
 		Engine->Update();
+
+		Poly::OutputQueue& OutputEventsQueue = Engine->GetOutputQueue();
+		while (!OutputEventsQueue.IsEmpty())
```

Comment:
We should add that logic for input handling to editor also.

Reply:
Need to know where it is handled in Editor.

Reply:
Ask @PiotrMoscicki

Reply:
Engine::Update is called here - Editor/Src/Managers/Project/EngineManager.cpp
You can get SDL from - Editor/Src/Widgets/Inspectors/ViewportInspectorWidget.cpp
You can write getter for ViewportInsepctor InspectorManager at- Editor/Src/Managers/Project/InspectorManager.cpp
and then get ViewportInsepctor using gApp (gApp->InspectorMgr)